scripts/common/preflights.sh

```diff
@@ -605,3 +605,26 @@ function host_preflights_mkresults() {
     chmod -R +r "${DIR}/${HOST_PREFLIGHTS_RESULTS_OUTPUT_DIR}/" # make sure the file is readable by kots support bundle
     rm -f "${tmp_file}"
 }
+
+# Uninstall Docker when containerd is selected to be installed and it is a new install
+# So that, is possible to avoid conflicts
+allow_remove_docker_new_install() {
+     # If docker is not installed OR if conatinerd is not in the spec
+     # then, the docker should not be uninstalled
+     if ! commandExists docker || [ -z "$CONTAINERD_VERSION" ]; then
+          return
+     fi
+
+     # if k8s is installed already then, the docker should not be uninstalled
+     # so that it can be properly migrated to containerd
+     if commandExists kubectl ; then
+          return
+     fi
+
+     printf "\n${YELLOW}Docker already exists on this machine and Kubernetes is not installed. In order to avoid conflicts when containerd will be installed do you allow Docker be uninstalled first?${NC} "
```

Comment:
```suggestion
     printf "\n${YELLOW}Docker already exists on this machine and Kubernetes is not installed. In order to avoid conflicts when installing containerd Docker needs to be removed. Continue?${NC} "
```

Reply:
After we merge https://github.com/replicatedhq/kURL/pull/3844 we might be able to fix in the biggest part of the scenarios.  So, because of this I think a more optional/recommended wording fits better here. Is that make sense?

Also, `Remove Docker?` is not gramarly wrong?